components/home/MostUsedFeatures.vue

```diff
@@ -65,69 +78,26 @@
 
 <script>
 export default {
+  props: {
+    heading: {
+      type: Array,
+      default: () => [],
+    },
```

Comment:
I see everywhere we named the variable as singular and the type is plural. Any specific reason for doing so.